### ISSUE TITLE
error executing wasm-pack new (cannot parse 'latest' as a semver)

### ISSUE THREAD

**Issue by laurogama** [rustwasm/wasm-pack#907]
## 🐛 Bug description
When running wasm-pack new rust-example an error in cargo-generate occurs
the following log is printed
```[INFO]: Installing cargo-generate...
error: the `--vers` provided, `latest`, is not a valid semver version: cannot parse 'latest' as a semver

Error: Installing cargo-generate with cargo
Caused by: failed to execute `cargo install`: exited with exit code: 101
  full command: "cargo" "install" "--force" "cargo-generate" "--version" "latest" "--root" "/home/lgama/.cache/.wasm-pack/.cargo-generate-cargo-install-latest"
```
#### 🤔 Expected Behavior

a new wasm-pack project should be created
#### 👟 Steps to reproduce
Have a previous rust toolchain installed
install wasm-pack with `curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh`
execute `wasm-pack new rust-example`

#### 🌍 Your environment

wasm-pack version: 0.9.1
rustc version: rustc 1.45.0 (5c1f21c3b 2020-07-13)

**Comment by laurogama** [comment]
similar to #828

**Comment by richiksc** [comment]
I get the same issue when running `wasm-pack new` v0.9.1 with the patched lockfile from #960 on an Apple Silicon based Mac.

**Comment by fasidOnGit** [comment]
Facing the same issue with `wasm-pack new`

**Comment by SamNelson** [comment]
A work around for this is to just `cargo install cargo-generate`.
If wasm-pack finds cargo-generate it will skip the install.

**Comment by fasidOnGit** [comment]
@SamNelson Thanks for the quick response, I was actually doing that. Just in case if it helps anyone I also had to do https://github.com/sfackler/rust-openssl/issues/1021#issuecomment-544779669 , before `cargo install cargo-generate` to make it successful.

**Comment by AnubhabB** [comment]
Facing the same issue on Mac wasm-pack v0.9.1. `cargo install cargo-generate` didn't help.

**Comment by fasidOnGit** [comment]
We are recently bitten by this error too often. A recent PR that updated the docs had the same issue,
https://github.com/rustwasm/wasm-pack/pull/967#issuecomment-775942728

**Comment by WanderHuang** [comment]
Same problem met in OSX. I use this method instead:

1. `cargo install cargo-generate`
2. `cargo generate --git https://github.com/ashleygwilliams/wasm-pack-template.git`

It just works to generate a crate by template

**Comment by AlexMikhalev** [comment]
@WanderHuang thank you for your comment - it helped me to progress. 
The issue isn't mac specific - the same behaviour on Linux and cargo install cargo-generate didn't help.

**Comment by raffaeler** [comment]
Same here on Windows.
An additional problem is that if you run the build command manually (`cargo install --force cargo-generate --root folder_name`) inside a developer command prompt (the one already having the environment variables set), the build fails with hundreds of `error LNK2019: unresolved external ` errors. It works if you start from a command prompt (no developer).

Once the cargo-generate.exe is created, you have to manually copy it in the `.cargo/bin` folder.
Finally the `wasm-pack new projectname` works, but hey!

**Comment by insinfo** [comment]
Facing the same issue with wasm-pack new example
wasm-pack 0.9.1

**Comment by drager** [comment]
Should be fixed in the [v0.10.0](https://github.com/rustwasm/wasm-pack/releases/tag/v0.10.0)
